layouts/partials/fragments/footer.html

```diff
@@ -1,10 +1,43 @@
-{{- $name := strings.TrimSuffix ".md" (replace .self.Name "/index.md" "") -}}
 {{- $bg := .Params.background | default "dark" }}
 
+{{/* Resource fallthrough path preparation */}}
+{{/* Do not change the following snippet */}}
+{{/* Code is duplicated throughout the code */}}
+{{- $path := .self.File.Path -}}
+{{- $path := replace $path "/index.md" "" -}}
+{{- $path := strings.TrimSuffix ".md" $path -}}
+{{- $page_path := replace $path (printf "/%s" .Name) "" -}}
+{{/* End of do not change */}}
+
+{{/* Global resource fallback - can also be used within loops */}}
+{{- $image := .Params.logo.image -}}
+{{/* Do not change the following snippet */}}
+{{/* Code is duplicated throughout the code */}}
+{{- $.root.Scratch.Set "image" (printf "images/%s" $image) -}}
+{{- range $.resources.ByType "image" -}}
```

Comment:
I think this loop is not achieving anything. It's adding an scope and running the code block as many times as there are resources in the page but it's achieving the results of the first loop just several times.

Reply:
Not sure I get the whole point. So you would just remove it?

Reply:
I've read the code several times and can't figure out how it's useful. Am I missing something?

Reply:
Nope I missed something. ^^